doc/develop/fixing_data.rst

```diff
@@ -305,3 +305,30 @@ strictness to the highest:
 - ``strict``: fail if there are any warnings, this is the highest level of
     strictness. Mostly useful for checking datasets that you have produced, to
     be sure that future users will not be distracted by inoffensive warnings.
+
+
+Natively supported non-CMOR datasets
+====================================
+
+Fixed datasets are supported through the ``native6`` project. Below is a list of
+datasets currently supported.
+
+ERA5
+----
+
+- Supported variables: ``clt``, ``evspsbl``, ``evspsblpot``, ``mrro``, ``pr``, ``prsn``, ``ps``, ``psl``, ``ptype``, ``rls``, ``rlds``, ``rsds``, ``rsdt``, ``rss``, ``uas``, ``vas``, ``tas``, ``tasmax``, ``tasmin``, ``tdps``, ``ts``, ``tsn`` (``E1hr``/``Amon``), ``orog`` (``fx``)
+- Tier: 3
+
+MSWEP
+-----
+
+- Supported variables: ``pr``
+- Supported frequencies: ``mon``, ``day``, `3hr``.
+- Tier: 3
+
+For example for monthly data, place the files in the ``/Tier3/MSWEP/latestversion/mon/pr`` subdirectory of your ``native6`` project location.
```

Comment:
Is this a general structure now for all datasets or specific to MSWEP? As in: do we need the same for ERA5? 

Also: would it make sense to use the actual version ("V220") instead of the latest version?

Reply:
I just put ERA5 there so the documentation is complete, I agree that this is best done in a separate PR for documentation fixes. Putting `latestversion` in the path always works, whereas putting `V220` may go out of date at some point.

This is not meant as a general structure, just something that should be complete for now. When more datasets will be supported, we can think about how to better organize the documentation there.